packages/lesswrong/components/localGroups/CommunityHome.tsx

```diff
@@ -129,7 +145,12 @@ const CommunityHome = ({classes}: {
             <SingleColumnSection>
               <SectionTitle title="Resources"/>
               <AnalyticsContext listContext={"communityResources"}>
-                <PostsList2 terms={{view: 'communityResourcePosts'}} showLoadMore={false} />
+                {isEAForum ?
+                  <Typography variant="body1">
+                      <a className={classes.link} href="https://eahub.org/groups?utm_source=forum.effectivealtruism.org&utm_medium=Organic&utm_campaign=Forum_Homepage">EA Hub Groups Directory</a>
+                  </Typography> :
+                  <PostsList2 terms={{view: 'communityResourcePosts'}} showLoadMore={false} />
+                }
```

Comment:
This is a liiiitle hacky. Maybe it'd be better to cut the Resources section altogether and put it in the description?

Reply:
Fixed in https://github.com/LessWrong2/Lesswrong2/pull/4102/commits/55211dac503049e38976d8ce1c8e29970a02f3b4.